docs/zh/latest/plugins/recaptcha.md

```diff
@@ -0,0 +1,112 @@
+---
+title: recaptcha
+---
+
+<!--
+#
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 描述
+
+通过向 Google reCAPTCHA 服务校验客户端传递的验证码来限制对上游服务的访问。插件支持自定义无效校验码的响应体。
+
+> 注意，此插件只支持 Google reCAPTCHA v2 版本。
+
+## 属性
+
+| Name      | Type          | Requirement | Default    | Valid                                                                    | Description                         |
+| --------- | ------------- | ----------- | ---------- | ------------------------------------------------------------------------ |-------------------------------------|
+| secret_key | string        | 必须    |            |  | Google reCAPTCHA v2 的 secret key    |
+| parameter_source | string | 可选 | header | | 验证码参数的来源枚举值。当前仅支持 `header`, `query` |
+| parameter_name | string | 可选 | captcha | | 验证码参数的名称                            |
+| response | object | 可选    | content_type  = `application/json; charset=utf-8`<br />status_code = `400`<br />body = `{"message":"invalid captcha"}` |  | 无效验证码的 HTTP 响应体                     |
```

Comment:
Why `<br>` here ?

Reply:
The response is an object type and it has three properties. And I just want to make sure it looks good. Has APISIX had any guidance on this? Or any suggestions from you?

Reply:
> Has APISIX had any guidance on this ?

Not yet.

> Or any suggestions from you ?

I'm worried that this might cause exceptions in other markdown renderers, shall we remove it ?

Reply:
Sure.